### DISCUSSION TITLE
splitting manga pages with "local" output files reversed, while still preserving overall page order

### DISCUSSION THREAD

**Discussion by alexjcode** [ImageMagick/ImageMagick#6595] [Digital Image Processing]
I'm splitting a bunch of manga pages that read R-to-L into 2 png files each, but I'd like to retain the original filenames and have the 2 output files named in reverse order while keeping the overall page order intact.

basically i want this result:
```
input:
001-002.png
output:
002.png (Left side), 001.png (Right side)

input:
003-004.png
output:
004.png (Left side), 003.png (Right side)
```

this is the code i have tried so far
```
magick *.png -scene 1 -crop 2x1@ +repage -quality 100 %03d.png
```
the result i'm getting is below. it names the pages incorrectly, so i have to go in and reorder them
```
input:
001-002.png
output:
001.png (Left side), 002.png (Right side) 

input:
003-004.png
output:
003.png (Left side), 004.png (Right side)
```

**Comment by snibgo** [comment]
I think this will work:
```
magick *.png -flop -scene 1 -crop 2x1@ +repage -flop -quality 100 %03d.png
```
`-flop` reverses each image left-right. So after the `-crop`, each right side image will be before the left side image. But each image will be reversed, so we need to `-flop` again.
